core/assets/bundles/bundle_vi.properties

```diff
@@ -1221,8 +1221,8 @@ block.copper-wall.name = Tường đồng
 block.copper-wall-large.name = Tường đồng lớn
 block.titanium-wall.name = Tường titan
 block.titanium-wall-large.name = Tường titan lớn
-block.plastanium-wall.name = Tường Nhựa
-block.plastanium-wall-large.name = Tường Nhựa lớn
+block.plastanium-wall.name = Tường Nhựa Kim Loại 
+block.plastanium-wall-large.name = Tường Nhựa Kim Loại lớn
```

Comment:
This is petty weird because plastic is not a metal...

Reply:
~~it's a titanium mixed oil (with create the plastic part~~
~~so it's partly metal make it cappable of absorbing laser~~